plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/utils/HuaweiMonitorClientUtil.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator.plugin.huaweicloud.monitor.utils;
+
+import com.huaweicloud.sdk.ces.v1.CesClient;
+import com.huaweicloud.sdk.ces.v1.region.CesRegion;
+import com.huaweicloud.sdk.core.auth.BasicCredentials;
+import com.huaweicloud.sdk.core.auth.ICredential;
+import com.huaweicloud.sdk.core.exception.SdkException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+/**
+ * HuaweiCloudMonitor Client Util.
+ */
+@Slf4j
+@Component
+public class HuaweiMonitorClientUtil {
+
+    /**
+     * Get Huawei Monitor ICredential by IAM.
+     *
+     * @param ak huawei ak.
+     * @param sk huawei sk.
+     */
+    public ICredential getIcredential(String ak, String sk) {
+        if (StringUtils.isBlank(ak) || StringUtils.isBlank(sk)) {
+            log.error("Get ICredential error,AK or SK is empty");
+            return null;
+        }
+        return new BasicCredentials()
+                .withAk(ak).withSk(sk);
+    }
+
+    /**
+     * Get HuaweiCloud Monitor Client.
+     *
+     * @param icredential ICredential.
+     */
+    public CesClient getCesClient(ICredential icredential, String regionName) {
+        CesClient client;
+        try {
+            client = CesClient.newBuilder()
+                    .withCredential(icredential)
+                    .withRegion(CesRegion.valueOf(regionName))
+                    .build();
+        } catch (SdkException e) {
+            log.error("Get HuaweiCloud Monitor CesClient failed.");
+            return null;
```

Comment:
should we throw an exception here? otherwise the calling method will have null pointers